group_test.go

```diff
@@ -384,24 +386,45 @@ func TestGroup(t *testing.T) {
 		{
 			name: "Static",
 			setup: func(req *http.Request) {
-				fiber.Static("static", "./")
+				tempDir, err := os.MkdirTemp("", "test")
+				assert.NoError(t, err)
+
+				err = os.WriteFile(filepath.Join(tempDir, "test.json"), []byte("{\"id\":1}"), 0755)
+				assert.NoError(t, err)
+
+				fiber.Static("static", tempDir)
 			},
-			method:     "GET",
-			url:        "/static/README.md",
-			expectCode: http.StatusOK,
+			method:         "GET",
+			url:            "/static/test.json",
+			expectCode:     http.StatusOK,
+			expectBodyJson: "{\"id\":1}",
 		},
 		{
 			name: "StaticFile",
 			setup: func(req *http.Request) {
-				fiber.StaticFile("static-file", "./README.md")
+				file, err := os.CreateTemp("", "test")
+				assert.NoError(t, err)
+
+				err = os.WriteFile(file.Name(), []byte("{\"id\":1}"), 0755)
+				assert.NoError(t, err)
+
+				fiber.StaticFile("static-file", file.Name())
 			},
-			method:     "GET",
-			url:        "/static-file",
-			expectCode: http.StatusOK,
+			method:         "GET",
+			url:            "/static-file",
+			expectCode:     http.StatusOK,
+			expectBodyJson: "{\"id\":1}",
 		},
 		{
 			name: "StaticFS",
 			setup: func(req *http.Request) {
+				tempDir, err := os.MkdirTemp("", "test")
+				assert.NoError(t, err)
+				defer os.RemoveAll(tempDir)
+
+				err = os.WriteFile(filepath.Join(tempDir, "test.json"), []byte("{\"id\":1}"), 0755)
+				assert.NoError(t, err)
+
 				fiber.StaticFS("static-fs", http.Dir("./"))
 			},
 			method:     "GET",
```

Comment:
The URL of next line is `/static-fs/README.md`, do we need to change it？

Reply:
Oh, we may don't need to change this test case, it's `http.Dir("./")`.

Reply:
Fixed